en/04_Changelogs/5.1.0.md

```diff
@@ -15,6 +15,7 @@ title: 5.1.0 (unreleased)
 ### Other new features
 - You can now exclude specific `DataObject` models from the check and repair step of `dev/build` - see [ORM Performance](/developer_guides/performance/orm/#skip-check-and-repair) for more information.
 - You can change what `SearchFilter` the `TreeDropdownField` uses with yaml configuration - see [ORM Performance](/developer_guides/performance/orm/#treedropdownfield) for more information.
+- Extensions which modify permissions for [`Member`](api:SilverStripe\Security\Member) records can now return `true` and have that be respected, the same as when modifying permissions for any other `DataObject` record.
```

Comment:
It's Group, not Member that has changed. Member already uses this behaviour.

Reply:
Ahh yes, thanks. Will fix.